core/modules/nfv_monitor.cc

```diff
@@ -134,6 +134,17 @@ void NFVMonitor::ProcessBatch(Context *ctx, bess::PacketBatch *batch) {
     //   it->second += 1;
     // }
 
+    uint32_t id = bess::utils::bucket_stats.rss_hash_to_id(reinterpret_cast<rte_mbuf*>(pkt)->hash.rss);
+    packet_counter_lock_.lock();
+    bess::utils::bucket_stats.bucket_table_lock.lock_shared();
```

Comment:
Your comment says: "Access within the same core is synchronized by packet_counter_lock."
Why do you still call lock here? Any cases where multiple NFVMonitor modules run on the same core?